src/pages/account/organization/new/form/form.hooks.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import { api } from '@tidb-community/datasource';
+import { getErrorMessage } from './form.utils';
+
+export const useCustomFormItems = () => {
+  const [validationErrors, setValidationErrors] = useState(() => ({}));
+
+  const formItemProps = (name) => {
+    if (name in validationErrors) {
+      return {
+        name,
+        validateStatus: 'error',
+        help: getErrorMessage(validationErrors[name]),
+      };
+    } else {
+      return { name };
+    }
+  };
```

Comment:
Please consider using [Formik](https://formik.org/) for front-end validation.

Reply:
I will redesign this at final, this is just a ui prototype now.